compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -208,9 +208,23 @@ def AMDAIEPad :
 
 def AMDAIEPeelForLoop :
     Pass<"iree-amdaie-peel-for-loop", "func::FuncOp"> {
-  let summary = "Pass to peel the first iteration out of the scf.for loop.";
+  let summary = "Pass to peel the first or/and the last iteration out of the scf.for loop.";
   let constructor =
       "mlir::iree_compiler::AMDAIE::createAMDAIEPeelForLoopPass()";
+  let options = [
+    Option<"peelLoopType", "peel-loop-type",
```

Comment:
Nit: `peel-loop-type` naming-wise suggests "type of the loop", to me.
How about `peeling-type` ?

This wouldn't matter in the greater scheme of work but I thought I should add a little nudge on this.